### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by Atefeh-Behroozi**
Finished my homework!

### CODE REVIEW

**Review by naresh699 — APPROVED** [review]
not sure why the first two files were created, either this has used in the script or just for a dummy test. 

Please go ahead and merge

File: javascript/javascript1/week1/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  console.log(`Your dog will be ${dogYear} human years old in ${dogYearFuture}`);
+}
+
+
+
+
+
+//A house price estimator
+// Calculation for Peter's house
+let width = 8;
```
[the remark this answers is not in the document]
Comment (Atefeh-Behroozi) [reply javascript/javascript1/week1/index.js:32]: Hi Naresh, thanks for your comment and Ok, I avoid variable name which reserved in JS or CSS.

File: javascript/javascript1/week1/index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+let depth = 10;
+let height = 10;
+let gardenSizeInM2 = 100;
+let volumeInMeters = width * depth * height;
+let expectedPrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+let actualPrice = 2500000;
+
+if (actualPrice > expectedPrice) {
+  console.log("Peter is paying too much.");
```
[the remark this answers is not in the document]
Comment (Atefeh-Behroozi) [reply javascript/javascript1/week1/index.js:41]: Ok, sure.

[the remark this answers is not in the document]
Comment (Atefeh-Behroozi) [reply javascript/javascript1/week1/index.js:41]: Sorry for my first two lines, it was just for dummy test.

**Pull request merged**